miner/worker.go

```diff
@@ -778,6 +768,11 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, txFe
 			log.Trace("Skipping account with hight nonce", "sender", from, "nonce", tx.Nonce())
 			txs.Pop()
 
+		case core.ErrGasPriceDoesNotExceedMinimum:
+			// We are below the GPM, so we can stop (the rest of the transactions will either have
+			// even lower gas price or won't be mineable yet due to their nonce)
```

Comment:
It might be useful to have a `Trace` log here
```suggestion
			// even lower gas price or won't be mineable yet due to their nonce)
			log.Trace("Skipping remaining transaction below the gas price minimum")
```

Reply:
Sounds good.  The gas price and GPM will have been logged as debug already (see state_transition.go change in this PR), so a message by itself here works.